data/transport missions.txt

```diff
@@ -60,15 +63,21 @@ mission "Vintage Spaceman"
 				`	(Point like there's someone famous behind him and make a run for the door.)`
 					decline
 	on complete
-		payment 10000
+		payment 100
 		conversation
 			`"We're here!" Vintage spaceman slams the emergency unlock and jumps out while your ship is still a meter off the ground. He runs off in to the crowd.`
-			`	You grab a drink, climb on top of the <ship> and relax.`
-			`	After the concert is over you see your passenger emerge from the crowd, his head hung low. "It's all a ruse man. She was just a bot. A marketing tool to get me to buy tickets."`
-			`	"Guess she was made for you." You quip, feeling a little bad about it.`
-			`	"Sorry I wasted your time. I'm such a sucker. I don't have any credits for you, but you can have this suit. I don't feel like wearing it any more."`
-			`	He leaves the suit on your deck and disappears back in to the crowd. You take the suit to a pawn shop and get <payment> for it.`
-
+				choice
+					`	(Grab a beverage, climb on top of the <ship> and relax)`
+					`	(Unload cargo)`
+				`	A while later you see your passenger emerge from the crowd, his head hung low. "It's all a ruse, man. She was just a bot. A marketing tool to get me to buy tickets."`
+				choice
+					`	"I'm sure there's a girl out there for you."`
+					goto end
+					`	(Rub it in)`
+				`	"Guess she was made for you." You quip, feeling a little bad about it.`
```

Comment:
I originally wanted to put "was" in italics to try and reference what the spaceman said before "It's like she was made for me." 

Also, is it common for all the dialogue to have strictly correct grammar? Many people talk like that, but I can see how it's confusing.